### ISSUE TITLE
Cannot connect: Login error: ERROR_QUEUE_OVERFLOW

### ISSUE THREAD

**Issue by thorgrin** [bitlbee/bitlbee-facebook#180]
Today, I started getting this error:
```
14:53:37 <@root> facebook - Logging in: Authenticating                                                                                                                                                                         
14:53:38 <@root> facebook - Logging in: Fetching contacts
14:53:39 <@root> facebook - Logging in: Connecting
14:53:40 <@root> facebook - Login error: ERROR_QUEUE_OVERFLOW                                                                                                                                                                  
14:53:40 <@root> facebook - Logging in: Signing off..                                                                                                                                                                          
14:53:40 <@root> facebook - Logging in: Reconnecting in 15 seconds..
```

I'm on Ubuntu 18.04.1 and the plugin is installed from its universe repo:
```
aptitude show bitlbee-plugin-facebook
Package: bitlbee-plugin-facebook
Version: 1.1.2-1
State: installed
Automatically installed: no
Priority: optional
Section: universe/net
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Architecture: amd64
Uncompressed Size: 154 k
Depends: bitlbee (>= 3.4) | bitlbee-libpurple (>= 3.4), libglib2.0-0 (>= 2.37.3), libjson-glib-1.0-0 (>= 0.14), libc6 (>= 2.14)
Provides: bitlbee-facebook
```

Is this a known problem that can be solved by compiling latest version from source, or something new?
Thanks you.

**Comment by liskin** [comment]
Seems related to https://github.com/bitlbee/bitlbee-facebook/issues/129#issuecomment-451075814

**Comment by thorgrin** [comment]
For the record, I've tried the code from development branch, but the problem is the same.

**Comment by mijofa** [comment]
I've been getting the same since ~9:30am Thursday (2 days & 3hrs ago) I'm running on a Debian Jessie system, I was running from the old jgeboski apt repo, but to try and fix this problem I compiled directly from the source code here and the symptoms didn't change at all.

**Comment by tgerczei** [comment]
Same here, for the last few days in a row now. Also tried re-compiling everything from master as well as purging any caches to no avail.

**Comment by NPKJI** [comment]
I did a debug-log and found several messages about upgrading to a new version of messenger:

```Thank you for using Messenger. We regret to inform you that starting today, March 29, the version of the app you're using will no longer be supported and will no longer allow you to send and receive messages. To continue using Messenger, please update your Messenger app to the latest version: https:\/\
/play.google.com\/store\/apps\/details?id=com.facebook.orca If your phone does not support the newest versions of Messenger, you have two options: 1) You can install Facebook Lite to access your messages. Facebook Lite is the lightweight version of Facebook includes messaging, is fast, uses less data and works in slower network conditions. You can install it
 from here: https:\/\/fb.me\/bTOyGFpIKf3j7N 2) We recommend going to https:\/\/m.facebook.com\/messages on your mobile browser. Thanks, The Messenger Team```

**Comment by thorgrin** [comment]
I've update user agent in `facebook-api.h` to 
```
#define FB_ORCA_AGENT "[FBAN/Orca-Android;FBAV/192.0.0.31.101;FBBV/14477681]"
```

And it connects, sends and receives messages. Maybe something else, like groupchats, is broken, but I can live with it as a workaround.

**Comment by thorgrin** [comment]
Should I close the issue or wait for the packages to come out?

**Comment by dequis** [comment]
I usually like to keep things open for a while for visibility. I do intend to make a new versioned release soonish.

**Comment by brobostigon** [comment]
i am using ubuntu 18.04 and just updated to "bitlbee-facebook (20190107~7682a35~323)", and still receive the "Login error: ERROR_QUEUE_OVERFLOW" error, so the above fix has not resolved it.

**Comment by brobostigon** [comment]
i restarted bitlbee, but did not reconnect, and reconnecting resolved it.

**Comment by dequis** [comment]
Released v1.2.0 with the fix for this

Note that the general error ("ERROR_QUEUE_OVERFLOW") can potentially have other causes unrelated to this issue so it might still appear here in the future.

If you're using v1.2.0 or higher and this happens, please open another issue instead of adding comments here.
